.gitpod.yml

```diff
@@ -0,0 +1,90 @@
+# Learn more about this file at https://www.gitpod.io/docs/references/gitpod-yml
+tasks:
+  - name: Service Server
+    # Get Web API urls for use in Next build
+    before: |
+      eval $(gp env -e GITPOD_API_URL=$(gp url 3000))
+      eval $(gp env -e GITPOD_WEB_URL=$(gp url 8000))
+      eval $(gp env -e GITPOD_MICROSERVICE_URL=$(gp url 8443))
+    init: |
+      npm install
+      docker-compose --env-file config/env.gitpod build
+    # GitPod workspace urls are not stable, we have to look up urls and build again
```

Comment:
If they aren't stable, is there any point in doing this twice?  Why not just do it once late in the process and use that value?